README.md

```diff
@@ -8,25 +8,35 @@ http://oss.oracle.com/licenses/upl.
 
 -----
 
-# Public Slack Channel Available
+# Oracle Coherence Operator
 
-Need more help? Have a suggestion? Come and say "Hello!"
+Oracle enables organizations using Coherence to move their clusters into the cloud. By supporting industry standards, such as Docker and Kubernetes, Oracle facilitates running Coherence on cloud-neutral infrastructure. In addition, Oracle provides an open-source Coherence Operator ("the operator"), which implements features to assist with deploying and managing Coherence clusters in a Kubernetes environment. You can:
 
-We have a **public Slack channel** where you can get in touch with us to
-ask questions about using the operator or give us feedback or
-suggestions about what features and improvements you would like to see.
-We would love to hear from you. To join our channel, please [visit this site to get an invitation](https://join.slack.com/t/oraclecoherence/shared_invite/enQtNjA3MTU3MTk0MTE3LWZhMTdhM2E0ZDY2Y2FmZDhiOThlYzJjYTc5NzdkYWVlMzUzODZiNTI4ZWU3ZTlmNDQ4MmE1OTRhOWI1MmIxZjQ).  The
-invitation email will include details of how to access our Slack
-workspace.  After you are logged in, please come to `#operator` and say,
-"hello!"
+* Run Coherence within the industry standard Kubernetes container orchestration framework, using Docker containers for the members of a Coherence cluster.
+* Use popular industry standard tools such as Grafana, ELK (or more specifically the Elasticsearch, Fluentd and Kibana (EFK) stack, and Prometheus to monitor the performance, logs, and health of your clusters.
+* Flexibly override and customize cluster configuration.
+* Scale the Coherence deployment.
```

Comment:
Using kubernetes verbs and/or updates, scaler the Coherence deployment.

Reply:
Can you explain more on the issue?

Reply:
I think Harvey means change the last bullet to say something like: "Scale the Coherence deployment using Kubernetes verbs and/or updates."  Or "Using Kubernetes verbs and/or updates, scale the Coherence deployment."

Reply:
fixed